File: Examples/DotNET/SDK/CSharp/Images/UpdateBmpPropertiesWithoutStorage.cs
//////////////////////////////////////////////////////////////////////////
// Copyright 2001-2015 Aspose Pty Ltd. All Rights Reserved.
//
// This file is part of Aspose.Imaging. The source code in this file
// is only intended as a supplement to the documentation, and is provided
// "as is", without warranty of any kind, either expressed or implied.
//////////////////////////////////////////////////////////////////////////

using System;
using Aspose.Cloud;

namespace Aspose.Imaging.Cloud.Examples.Images
{
    class UpdateBmpPropertiesWithoutStorage
    {
        static void Main()
        {
            string dataDir = Aspose.Imaging.Cloud.Examples.Common.GetDataDir(System.Reflection.MethodBase.GetCurrentMethod().DeclaringType);
            string input = "sample1.bmp";
            string output = "output.bmp";
            
            int bitsPerPixel = 24;
            int horizontalResolution = 300;
            int verticalResolution = 300;
            
            Common.GetImagingSdk().Bmp.UpdateParametersOfBmpImage(
                bitsPerPixel,
                horizontalResolution,
                verticalResolution,
                dataDir + output,
                dataDir+input
                );
        }
    }
}

File: Examples/PHP/SDK/images/UpdateBmpPropertiesWithoutStorage.php
<?php require "../common.php";

$inputName = "sample1.bmp";
$inputPath = getPath(__FILE__, $inputName);
$outputName = "output.bmp";

$bitsPerPixel = 200;
$horizontalResolution = 100;
$verticalResolution = 50;
$fromScratch = false;

$outputPath = getPath(__FILE__, $outputName);

//getStorageSdk()->PostImageBmp($bitsPerPixel, $horizontalResolution, $verticalResolution, $fromScratch, $outputPath, $inputPath);
$response = getImagingSdk()->PostImageBmp($bitsPerPixel, $horizontalResolution, $verticalResolution, $fromScratch, $outputPath, $inputPath);

print_r($response);


